src/ConstructiveSolidGeometry/VolumePrimitives/Cone.jl

```diff
@@ -156,8 +156,8 @@ function surfaces(t::Cylinder{T}) where {T}
     bot_center_pt = _transform_into_global_coordinate_system(CartesianPoint{T}(zero(T), zero(T), -t.hZ), t) 
     top_center_pt = _transform_into_global_coordinate_system(CartesianPoint{T}(zero(T), zero(T), +t.hZ), t) 
     mantle = FullConeMantle{T,:inwards}((t.r, t.r), t.φ, t.hZ, t.origin, t.rotation)
-    e_bot = CircularArea{T}(r = t.r, φ = t.φ, origin = bot_center_pt, rotation = t.rotation)
-    e_top = CircularArea{T}(r = t.r, φ = t.φ, origin = top_center_pt, rotation = -t.rotation * RotZ{T}(π))
+    e_bot = EllipticalSurface{T}(r = t.r, φ = t.φ, origin = bot_center_pt, rotation = t.rotation)
```

Comment:
Are we still using the aliases `CircularArea` etc. somewhere?
If not, we might think about deleting them (?) from [lines 66-70 in EllipticalSurface.jl](https://github.com/SebastianRuffert/SolidStateDetectors.jl/blob/b09c61e4cbb952a39e64aa02b793669ba4fa3dc2/src/ConstructiveSolidGeometry/SurfacePrimitives/EllipticalSurface.jl#L66)

Reply:
![image](https://user-images.githubusercontent.com/88187639/170018342-02091f64-d928-418b-90b0-7e44c7eda0d9.png)